doc/book/getting-started/skeleton.md

```diff
@@ -44,12 +44,63 @@ $ composer serve
 This starts up a web server on localhost port 8080; browse to
 http://localhost:8080/ to see if your application responds correctly!
 
+> ### Setting a timeout
+>
+> Composer commands time out after 300 seconds (5 minutes). On Linux-based
+> systems, the `php -S` command that `composer serve` spawns continues running
+> as a background process, but on other systems halts when the timeout occurs.
+>
+> If you want the server to live longer, you can use the
+> `COMPOSER_PROCESS_TIMEOUT` environment variable when executing `composer
+> serve` to extend the timeout. As an example, the following will extend it
+> to a full day:
+>
+> ```bash
+> $ COMPOSER_PROCESS_TIMEOUT=86400 composer serve
```

Comment:
First time I read your comment I wanted to reply and say I wouldn't want to advise setting it in the config. But now that I read it again, you set it in the global config, which affects only your local dev environment. I actually like this a lot more than other solutions and I think it's cleaner.

However I think 1 day is a lot. What do you think, lower it to a few hours? You need a break once in a while any way.